xml/security_ldap_modules.xml

```diff
@@ -37,19 +37,19 @@ ACL Plugin
 ACL preoperation
 [...]</screen>
   <para>
-    The following command enables the MemberOf plugin referenced in
-    <xref linkend="sec-security-ldap-server-sssd"/>. MemberOf simplifies
+    The following command enables the <literal>MemberOf</literal> plug-in referenced in
```

Comment:
389-ds calls them 'plugins' so shourld we stay consistent?

Reply:
According to the standards defined in our SUSE style guide for product documentation, the spelling within our product docs is 'plug-ins', see https://documentation.suse.com/style/current/single-html/docu_styleguide/#sec-terminology

Which means as long as I do not cite from the code or UI, I need to stick to our officially agreed spelling (which in this case differs from what 389-ds uses...).

Reply:
In this case wouldn't it be a "reference" to the memberof plugin in the code though ?

Reply:
Looking at the `screen` elements in l. 47 and l. 73, I'm wondering now, too. Let's ask @dariavladykina, our editor.

Reply:
@Firstyear : I discussed this with our editor and she sees it the same way like me. Unless it is a direct quote from the code, the UI (or a command), we should use 'plug-in'. I also checked the RH docs and they actually treat it the same way. 

See for example https://access.redhat.com/documentation/en-us/red_hat_directory_server/11/html/administration_guide/tracking_the_bind_dn_for_plug-in-initiated_updates or https://access.redhat.com/documentation/en-us/red_hat_directory_server/11/html/administration_guide/managing_directory_entries#renaming-users-groups-posix-groups-and-ous (scroll two paras up from the heading '3.1.6.2. Renaming Users, Groups, POSIX Groups, and OUs' to find the examples).